examples/todomvc/src/app.container.js

```diff
@@ -0,0 +1,11 @@
+import { provider, inject } from 'k-simple-state/react'
+import { compose } from 'recompose'
+import loader from 'hoc-react-loader'
+import store from './store'
+import App from './app'
+
+export default compose(
+  provider(store),
+  inject(st => ({ load: () => { st.dispatch({ type: '@@ui/APP_LOADED' }) } })),
```

Comment:
This kind of event could be autmatically dispatched by the `provider` hoc when the store is provided.
an action like `@@kst/STORE_PROVIDED`,  `@@kst/STORE_LOADED` or  `@@kst/INIT`

I can create an issue if you think it's a good idea

Reply:
i vote for `@@kst/INIT` !
Good idea @bpetetot  <3

Reply:
Yeah you should create an issue :)
And yes this is a good idea